### DISCUSSION TITLE
TextureView no longer transparent after updating to Filament version v1.42.2

### DISCUSSION THREAD

**Discussion by sebapian** [google/filament#7220] [Q&A]
As per the discussion title, previously working TextureView from v1.34.0 was transparent in between background image and 3D model (sandwiched). Code is below:
```
private fun setBackgroundTransparent() {
        updateFog(enabled = false)
        textureView.isOpaque = false
        filamentView.scene?.skybox = null
        // clear the swapchain with transparent pixels
        val options = renderer.clearOptions
        options.clear = true
        renderer.clearOptions = options
    }
```

However, after updating Filament to the latest version (while updating minimum API to 24), a black background is shown instead of a transparent one. This was tested on Pixel 4 and 5 emulators. Setting FilamentView blendmode to TRANSLUCENT also did not work. Do you know what may be causing this issue and what would be a suitable fix?

EDIT: Confirmed that change from version 1.34.0 to 1.35.0++ causes the issue.

**Comment by koanarec** [comment]
I am having this exact problem, its really annoying and I can't figure out a workaround. ? Any help would be great

**Comment by sebapian** [comment]
Hi there @romainguy , any chance looking at this issue? Could provide more information if required.
